### ISSUE TITLE
!say drops leading space in string.

### ISSUE THREAD

**Issue by Aquaricat** [UnicornHeartClub/Issues#77]
```
!roll 1d20 + @me.proficiency_bonus + @me.dexterity_mod + 2 >> $atk1 = $1
!roll 1d20 + @me.proficiency_bonus + @me.dexterity_mod + 2 >> $atk2 = $2
!say "First attack was " $atk1 " and second was " $atk2 "."
```
So here's a basic macro.  The expected output was a line displaying both `$atk1` and `$atk2`, and I did get that, *but*...

![image](https://user-images.githubusercontent.com/7004392/34804508-9fa5ec1e-f63e-11e7-9029-517a57154478.png)
Note the lack of a space between `$atk1` and `and` - the parser stripped out the leading space.

**Comment by tlackemann** [comment]
This should be fixed in 0.4.5

**Comment by Pjb518** [comment]
Issue is corrected, however, a new bug has been introduced in the process. The "." now adds a preceding space, though this is less of an issue, and is far less noticeable.

**Comment by Aquaricat** [comment]
![image](https://user-images.githubusercontent.com/7004392/34834488-b0b2afbe-f6b7-11e7-86a5-bca6fd48f4e6.png)
Can confirm the first fix, leading to the second bug.  ^^

**Comment by tlackemann** [comment]
Fixed in 0.5.2 :)

**Comment by Aquaricat** [comment]
Fix confirmed, closing.  ^^
